packaging/datadog-agent/source/supervisor.conf

```diff
@@ -19,7 +19,6 @@ serverurl = unix://%(here)s/../run/agent-supervisor.sock
 
 [program:collector]
 command=python agent/agent.py foreground --use-local-forwarder
-stdout_logfile=supervisord/logs/collector.log
```

Comment:
@remh Actually commit 2aca54b (in this PR) sets up `datadog.conf` in `setup_agent.sh` so that all the components of the agent log to the `logs/` directory, instead of using supervisor to redirect the stdout.

This could be an issue however if the new script is used on a machine that already has a `datadog.conf` file created by the old script. In that case the `datadog.conf` file would remain untouched by `setup_agent.sh` and we'd end up with no logs from the agent.

Do you think this case is worth changing what the script currently does?